tracks/ruby/exercises/high-scores/mentoring.md

```diff
@@ -0,0 +1,71 @@
+New concepts: Array query methods (+ chaining methods), instantiating a class, getter method or `attr_reader`.
+
+### Minimal solution for approval
+
+```ruby
+
+# version 2+
+
+class HighScores
+  def initialize(scores)
+    @scores = scores
+  end
+
+  def scores  # no need to change getter method to `attr_reader`
```

Comment:
Should this comment really be in the _solution_? IMO it's better as a block of text below this with an explanation _why_ it should not need to change to `attr_reader`. I think in this case you meant something along the lines of "it's fine to use a method instead of `attr_reader`, but it's not clear to me at all.